.github/workflows/autobuild.yml

```diff
@@ -205,22 +205,22 @@ jobs:
           - config_name:            Linux .deb amd64 (artifacts+codeQL)
             target_os:              linux
             building_on_os:         ubuntu-22.04
-            building_container:     ubuntu:18.04
+            building_container:     ubuntu:20.04
             base_command:           ./.github/autobuild/linux_deb.sh
             run_codeql:             true
             is_main_build_target:   true
 
           - config_name:            Linux .deb armhf (artifacts)
             target_os:              linux
             building_on_os:         ubuntu-22.04
-            building_container:     ubuntu:18.04
+            building_container:     ubuntu:20.04
             base_command:           TARGET_ARCH=armhf ./.github/autobuild/linux_deb.sh
             run_codeql:             false
 
           - config_name:            Linux .deb arm64 (artifacts)
             target_os:              linux
             building_on_os:         ubuntu-22.04
-            building_container:     ubuntu:18.04
+            building_container:     ubuntu:20.04
```

Comment:
I think this should be tested on older Ubuntu versions. We had problems with pan pots not showing up due to something being broken with the dependencies. 

@hoffie do you remember?

Reply:
I think we would be justified in specifying that the next version requires Ubuntu 20.04 or newer. I can easily check that the CI build runs on 20.04. I will need to create a 22.04 VM on VirtualBox before I can check it on that version.

I don't think we need to continue supporting 18.04.

Reply:
Hmmm, it looks like VirtualBox only provides 44100Hz audio to the guest OS, and not 48000Hz. So it might need a physical PC to test it. Maybe USB passthrough from host (Windows) to guest (Ubuntu) might allow a 48kHz audio device?

Reply:
OK, I managed to set up jack with a dummy device at 48000Hz, which allowed me to run the Jamulus client.

On Ubuntu 22, it does indeed fail to display the pan pot. Will try 20 as well.

Reply:
My mistake. I had not set the audio to Stereo. Once I had done so, the pan pot showed up fine on Ubuntu 22. Will try 20 too.

Reply:
OK, the artifact built on Ubuntu 20 from this PR's action runs fine on Ubuntu 20 and Ubuntu 22, and shows the pan pots correctly in both cases.

It does not run on Ubuntu 18, giving the following errors on invocation:
```
tony@ubuntu18desktop:~$ jamulus 
jamulus: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by jamulus)
jamulus: /usr/lib/x86_64-linux-gnu/libQt5Core.so.5: version `Qt_5.12' not found (required by jamulus)
```

I think this is fine. but we should make it clear somewhere that our future release builds need to run on Ubuntu 20 or newer.

Reply:
Done some further tests with the same artifact on different versions of Debian:
- Debian 12 (stable) (Qt 5.15.8): runs fine, pan pots show properly
- Debian 11 (oldstable) (Qt 5.15.2): runs fine, pan pots show properly
- Debian 10 (obsolete) (Qt 5.11.3): fails to run with the same errors as Ubuntu 18 above.

Reply:
Ok. So this sounds good.